frontend/src/Components/Cart/CartCard.jsx

```diff
@@ -0,0 +1,106 @@
+import axios from "axios";
+import { useState } from "react";
+import { BACKEND } from "../../api";
+import { RestApiCalls } from "../../utils/CallRestApi";
+import { useAuth } from "../../Context/AuthContext";
+import { useProduct } from "../../Context/ProductContext";
+import { calculateOriginalPrice, isInWishlist } from "../CardCommonFunctions";
+import { Modal } from "../Modal/Modal";
+import "./CartCard.css"
+
+
+export const CartCard = ({_id, product, quantity, isInCart}) => {
+    const {image,brandName,description,price,discountByPercentage,seller} = product;
+    const {state,dispatch} = useProduct();
+    const {auth} = useAuth();
+    const hideToast = () => {
+        setTimeout(() => {
+            dispatch({type:"TOGGLE_TOAST",payload:"", value: false});
+          }, 1000)
+    }
```

Comment:
This could go into utils or within the toast component.

Reply:
done